pages/auth/signin.js

```diff
@@ -0,0 +1,63 @@
+import BlankLayout from "@components/layouts/BlankLayout"
+import { getServerSession } from "next-auth/next"
+import { getProviders, signIn } from "next-auth/react"
+import { authOptions } from "pages/api/auth/[...nextauth]"
+import { BsGithub } from "react-icons/bs"
+
+export default function SignIn({ providers }) {
+
+    const handleProviderIcon = (provider_name) => {
+        if (provider_name === "GitHub") {
```

Comment:
A `switch` would be better here and then add a `default` at the end for a fallback login icon

Reply:
But for now maybe just add a return out side of the `if` with a default key/secure icon

Reply:
You mean like this?  

```js
const handleProviderIcon = (provider_name) => {
        if (provider_name === "GitHub") {
            return <BsGithub className="text-xl" />
        }

        return <LockIcon />
    }
```

Reply:
yes 👍 , but you might want to add the same class to it also